frontend/src/components/my-grant-profile-page-handler/my-grant.tsx

```diff
@@ -0,0 +1,225 @@
+import {
+  Button,
+  Grid,
+  GridColumn,
+  Link,
+  Spacer,
+  Text,
+  VerticalStack,
+} from "@python-italia/pycon-styleguide";
+import { FormattedMessage } from "react-intl";
+
+import { useCurrentLanguage } from "~/locale/context";
+import { DeadlineStatus, Status as GrantStatus } from "~/types";
+import type { MyProfileWithGrantQuery } from "~/types";
+
+import { useCountries } from "~/helpers/use-countries";
+import { createHref } from "../link";
+import { Sidebar } from "./sidebar";
+
+type Props = {
+  grant: MyProfileWithGrantQuery["me"]["grant"];
+  deadline: MyProfileWithGrantQuery["conference"]["deadline"];
+};
+
+export const MyGrant = ({ grant, deadline }: Props) => {
+  const language = useCurrentLanguage();
+  const countries = useCountries();
+
+  const canManageGrant = [
```

Comment:
I would move the array definition outside so it doesn't create a new array every render